.gitlab-ci.hpc-spack.yml

```diff
@@ -0,0 +1,22 @@
+include:
+  - project: hpc/gitlab-pipelines
+    file: spack-build-components.gitlab-ci.yml
+
+spack_setup:
+  extends: .spack_setup_ccache
+  variables:
+    SALLOC_ACCOUNT: proj3
+    SBATCH_ACCOUNT: proj3
+    SLURM_ACCOUNT: proj3
+    bb5_account: proj3
+    SPACK_PACKAGE_NAME: brayns
+    SPACK_PACKAGE_COMPILER: gcc
+    SPACK_PACKAGE_SPEC: ^ospray@1.8.5%intel +brion
```

Comment:
I think that these 3 variables should be defined on a file-global level, as both setup and build need them

Reply:
Changed, thanks!